frontend/src/components/ScenarioCard.tsx

```diff
@@ -83,6 +83,24 @@ export default function ScenarioCard(props: ScenarioCardProps): JSX.Element {
     return 'N/A';
   };
 
+  const getArrow = (compartment: string): JSX.Element => {
+    // shows downwards green arrows if getCompartmentRate < 0%
+    if (parseFloat(getCompartmentRate(compartment)) < 0) {
+      return <ArrowDropDownIcon color={'success'} fontSize={'medium'} sx={{display: 'block'}}></ArrowDropDownIcon>;
+    }
+    // shows upwards red arrows if getCompartmentRate > 3%. If there is no RKI value for that compartment i.e., getCompartmentRate is Null, then it will check the getCompartmentValue (scenario values only) which will always me positive.
```

Comment:
```suggestion
    // Shows upwards red arrows if getCompartmentRate > 3%. If there is no RKI value for that compartment i.e., getCompartmentRate is Null, then it will check the getCompartmentValue (scenario values only) which will always be positive.
```